### PR TITLE
added Websitepulse.com

### PR DESCRIPTION
**Pull request by peterwsp**
Added Websitepulse.com monitoring service and free test tools.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -412,6 +413,7 @@ Table of Contents
   * [statuscake.com](https://www.statuscake.com/) — Website monitoring, unlimited tests free with limitations
   * [thousandeyes.com](https://www.thousandeyes.com/) — Network and user experience monitoring. 3 locations and 20 data feeds of major web services free
   * [uptimerobot.com](https://uptimerobot.com/) — Website monitoring, 50 monitors free
+  * [websitepulse.com](https://www.websitepulse.com/) — Free basic uptime monitoring after a 30-day free full-service trial.
```
Comment (ripienaar) [inline README.md]: please list it once only

Comment (peterwsp) [reply README.md]: ok, new commit added.

**Comment by peterwsp** [comment]
removed Websitepulse.com duplicate - set only on "Tools" section.

**Comment by ripienaar** [comment]
thanks

**Pull request merged**